lib/types/Global.ts

```diff
@@ -1,5 +1,5 @@
 import { Backend } from "../core/backend";
-import { Kuzzle } from "../kuzzle";
+import { Kuzzle } from "../kuzzle/kuzzle";
```

Comment:
(If that doesn't break anything), maybe convert the [index.js](https://github.com/kuzzleio/kuzzle/blob/4ef12c7654f2a9af437054ebbae0324a8d481ea0/lib/kuzzle/index.js) file to TypeScript instead?